src/AutoBuild.jl

```diff
@@ -1268,80 +1275,98 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
         print(io, jll_jl)
     end
 
-    product_names(p::ExecutableProduct) = p.binnames
-    product_names(p::FileProduct) = p.paths
-    product_names(p::LibraryProduct) = p.libnames
-    product_names(p::FrameworkProduct) = p.libraryproduct.libnames
+    is_yggdrasil = get(ENV, "YGGDRASIL", "false") == "true"
+    ygg_head = is_yggdrasil ? LibGit2.head("..") : ""
+    src_name_first = uppercase(first(src_name))
+    print_source(io, s::ArchiveSource) = println(io, "* compressed archive: ", s.url, " (SHA256 checksum: `", s.hash,"`)")
+    print_source(io, s::GitSource) =     println(io, "* git repository: ", s.url, " (revision: `", s.hash,"`)")
+    print_source(io, s::FileSource) =    println(io, "* remote file: ", s.url, " (SHA256 checksum: `", s.hash,"`)")
+    function print_source(io, s::DirectorySource)
+        print(io, "* files in local directory: ")
```

Comment:
This is IMHO confusing: if I look at https://github.com/giordano/libsodium_jll.jl and read it there, I'd wonder "but where is this `./bundled` directory, I don't see it?".

Of course it's a bit better if this is from yggdrasil, but even then, "local directory" just doesn't make sense in my head (which might just indicate my head is too soft ;-) ). But how about this
```suggestion
        print(io, "* files bundled with the script used to generate this JLL: ")
```

Reply:
> "but where is this ./bundled directory, I don't see it?".

The link is broken because the example is totally fake

Reply:
I tried to clarify this, but with a slightly less verbose wording